docs/user/custom-domains.rst

```diff
@@ -0,0 +1,138 @@
+Custom Domains
+==============
+
+With custom domains you can serve your documentation from your own domain.
+By default your documentation is served from a Read the Docs :ref:`subdomain <hosting:subdomain support>`:
+``<slug>.readthedocs.io`` or ``<slug>.readthedocs-hosted.com``,
+for example ``https://pip.readthedocs.io``.
+
+.. contents::
+    :local:
+
+Adding a custom domain
+----------------------
+
+To setup your custom domain, follow these steps:
+
+#. Go the :guilabel:`Admin` tab of your project.
+#. Click on :guilabel:`Domains`.
+#. Enter your domain.
+#. Mark the :guilabel:`Canonical` option if you want use this domain
+   as your :doc:`canonical domain </canonical-urls>`.
+#. Click on :guilabel:`Add`.
+#. At the top of the next page you'll find the value of the CNAME record that you need to point your domain to.
+   For |org_brand| this is ``readthedocs.io``, and for :doc:`/commercial/index`
+   the record is in the form of ``<hash>.domains.readthedocs.com``.
+
+   .. note::
+
+      For a subdomain like ``docs.example.com`` add a CNAME record,
+      and for a root domain like ``example.com`` use an ANAME or ALIAS record.
+
+By default, we provide a validated SSL certificate for the domain,
+this service is provided by Cloudflare.
+The SSL certificate issuance can take about one hour,
+you can see the status of the certificate on the domain page in your project.
+
+As an example, fabric's DNS record looks like this:
+
+.. prompt:: bash $, auto
+
+   $ dig CNAME +short docs.fabfile.org
+     readthedocs.io.
+
+.. warning::
+
+   We don't support pointing subdomains or root domains to a project using A records.
+   DNS A records require a static IP address and our IPs may change without notice.
+
+Strict Transport Security
+-------------------------
+
+By default, we do not return a `Strict Transport Security header`_ (HSTS) for user custom domains.
+This is a conscious decision as it can be misconfigured in a not easily reversible way.
+For both |org_brand| and |com_brand|, HSTS for custom domains can be set upon request.
+
+We always return the HSTS header with a max-age of at least one year
+for our own domains including ``*.readthedocs.io``, ``*.readthedocs-hosted.com``, ``readthedocs.org`` and ``readthedocs.com``.
+
+.. _Strict Transport Security header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
+
+Legacy domains on |com_brand|
+-----------------------------
+
+Some older setups configured a CNAME record pointing to
+``<organization-slug>.users.readthedocs.com``,
+these domains will continue to resolve.
+
+Previously you were asked to add two records,
+this process has been simplified.
+If you have doubts about deleting some of the old records,
+please reach out to :ref:`support <support:user support>`.
```

Comment:
I'm not sure if this section adds some value to the document. We are not using this anymore and I think it's just confusing adding it here, in the middle of the document.

I'm sure if there are people with doubts about this, they will contact us without reading this section. If it's still working and they don't need to do anything or care about it, I'd remove it.

Reply:
👍  on removing this section. It should just be a single sentence if anything.